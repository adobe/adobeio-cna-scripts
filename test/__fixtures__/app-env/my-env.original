AIO_runtime_namespace=my-remote-namespace
AIO_runtime_auth=my-remote-auth
AIO_runtime_apihost=my-remote-apihost
