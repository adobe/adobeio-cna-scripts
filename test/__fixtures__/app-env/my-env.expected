AIO_RUNTIME_NAMESPACE=local-namespace
AIO_RUNTIME_AUTH=local-auth
AIO_RUNTIME_APIHOST=local-apihost
